extensions/amp-brid-player/0.1/test/test-amp-brid-player.js

```diff
@@ -22,8 +22,8 @@ describes.realWin(
       timer = Services.timerFor(win);
     });
 
-    function getBridPlayer(attributes, opt_responsive, config) {
-      const bc = doc.createElement('amp-brid-player');
+    function getBridPlayer(attributes, opt_responsive, config, alias = false) {
+      const bc = alias ? doc.createElement('amp-target-video-player') : doc.createElement('amp-brid-player');
```

Comment:
Please fix the linter error: https://app.circleci.com/pipelines/github/ampproject/amphtml/31401/workflows/2d856bf9-e6c5-427c-a21a-ffa0dbf30070/jobs/684888?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary

Reply:
Fixed!